tests/functional/test_integrity.py

```diff
@@ -27,3 +27,26 @@ def test_data_integrity_20s():
             run_workload(disk.system_path, runtime)
         with TestRun.step("Stop tracing"):
             iotrace.stop_tracing()
+
+
+def test_data_integrity_restart_tracing():
+    TestRun.LOGGER.info("Testing data integrity with multiple trace starts")
+    iotrace: IotracePlugin = TestRun.plugins['iotrace']
+    for disk in TestRun.dut.disks:
```

Comment:
We have to be sure we don't touch rootfs disk

Reply:
This code looks safe. This validation should be done by test-framework. This code only use rootfs disk when test-framework incorrectly recognizes it (right now it does that for rootfs on LVM partition). We could add some extra checks here, but I don't think it's necessary.